src/promptflow-core/tests/core/unittests/test_metrics.py

```diff
@@ -0,0 +1,49 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import pytest
+from promptflow.core._serving.monitor.metrics import MetricsRecorder
+from logging import logger
```

Comment:
this should be the line that breaks the CI for now (which can also break your local test as it's import)